### PR TITLE
Create CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by FMCalisto**
Fixes # .

### Description
A few sentences describing the changes proposed in this pull request.

### Status
**Ready/Work in progress/Hold**

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Integration tests passed locally by running `./runtests.sh --codeformat --coverage`.
- [ ] Quick tests passed locally by running `./runtests.sh --quick`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated, tested `make html` command in the `docs/` folder.

### CODE REVIEW

**Comment by wyli** [comment]
thanks! could you sign the commit? also please fill the doc with these meta data if needed: https://github.com/Project-MONAI/MONAI/blob/master/setup.cfg#L2

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+when an individual is representing the project or its community. Examples of
+representing a project or community include using an official project e-mail
+address, posting via an official social media account, or acting as an appointed
+representative at an online or offline event. Representation of a project may be
+further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the project team at . All
```
Comment (rijobro) [inline CODE_OF_CONDUCT.md:58]: Thanks. I guess an address is missing here, although I'm not sure what it should be. @wyli ?

Comment (FMCalisto) [reply CODE_OF_CONDUCT.md:58]: You should put the same address of the @Project-MONAI account.

**Review by wyli — APPROVED** [review]
this pr adds a default code of conduct document based on  https://github.com/github/opensource.guide#readme

**Pull request merged**